### ISSUE TITLE
Keyboard shortcuts to move focus between main libraryPane sections?

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4572]
Now that almost every important element is focusable, it can take a while to get from one side of the window to another. Maybe we should add a series of shortcuts to move focus between important components, allowing one to skip a lot of "tab" keypresses. 

https://forums.zotero.org/discussion/116878/feature-request-keyboard-shortcuts-to-better-navigate-panes-and-tabs
https://forums.zotero.org/discussion/114816/ux-and-keyboard-accessibility

Maybe a Tab with a modifier key would move focus forward but not to the next focusable element but to the next section. Just need to experiment and decide which modifier key would work.

**Comment by abaevbog** [comment]
Writing https://forums.zotero.org/discussion/comment/479863/#Comment_479863, I see that getting to the sidenav is also tricky and takes a lot of steps. As another idea, we could add a submenu to View, where main components of the interface would be listed and selecting the component focuses it

**Comment by matebitte** [comment]
To keyboard loving users looking at this and wondering "why is there no shortcut?":

You can workaround this missing feature by using the [Action Tags Plugin](https://github.com/windingwind/zotero-actions-tags/) and using [this little script](https://github.com/windingwind/zotero-actions-tags/discussions/606). 

Regarding actually solving this issue: I can hack around, but a proper feature is beyond my capabilities.
